README.org

```diff
@@ -191,7 +191,7 @@ These selectors take one argument alone, or multiple arguments in a list.
 +  =:category= :: Group items that match any of the given categories.  Argument may be a string or list of strings.
 +  =:children= :: Select any item that has child entries.  Argument may be ~t~ to match if it has any children, ~nil~ to match if it has no children, ~todo~ to match if it has children with any to-do keywords, or a string to match if it has children with certain to-do keywords.  You might use this to select items that are project top-level headings.  Be aware that this may be very slow in non-daily/weekly agenda views because of its recursive nature.
 +  =:date= :: Group items that have a date associated. Argument can be =t= to match items with any date, =nil= to match items without a date, or =today= to match items with today’s date. The =ts-date= text-property is matched against.
```

Comment:
Should other selectors, like `:date`, also support this?

Reply:
> Thanks for your patience. I think this shouldn't need much more before it can be merged.

Thanks for the review!
 
> Also, would you add tests for this, please? (i.e. in the test suite, not just that Org file)

Just added a quick test (not pushed yet). It fails, I guess that org-read-date is not aware of org-super-agenda-test-date so relative date specification can not be tested without further mocks. Do you have some suggestion on how to do that?

:date does not support arbitrary dates yet (only static: any, none, today), so relative ones are a double step!

Reply:
> > Thanks for your patience. I think this shouldn't need much more before it can be merged.
> 
> Thanks for the review!

FYI, feel free to ping me more often in the future.  I didn't intend to ignore this for a year; I just hadn't thought about working on org-super-agenda for a while, since I didn't need to change anything in it for my own sake, and I have a number of other projects that occupy my free time.

> > Also, would you add tests for this, please? (i.e. in the test suite, not just that Org file)
> 
> Just added a quick test (not pushed yet). It fails, I guess that org-read-date is not aware of org-super-agenda-test-date so relative date specification can not be tested without further mocks. Do you have some suggestion on how to do that?

I guess you'll have to look in the Org source code, find the appropriate function, and redefine it locally in the test using `cl-letf`.

> :date does not support arbitrary dates yet (only static: any, none, today), so relative ones are a double step!

As you can tell, I haven't worked on this package for a while.  :)  We should probably fix that too, though maybe not in this PR.